README_INTERNAL.md

```diff
@@ -1,9 +1,6 @@
 ## Releasing
 
-1. Update CHANGELOG.md
-1. Update version in package.json
+1. Run `npm version` which will bump the version in package.json and make a tag
+1. Push tags `git push --tags origin`
 1. Open Pull Request, merge after approved
-1. Tag master: example: `git tag v2.6.0` and `git push --tags origin`
-1. Run `npm publish`
-
-If releaseing a beta tag, run `npm publish --tag beta`
+1. Create a new release in the Github UI, give the release a name and add release notes (creating the release will kick off npm publish)
```

Comment:
The only addition I might make here would be to say that you need to make sure to select the tag created in the previous step when creating the new release, but otherwise this looks great! Do we want to include a template of what a release body should look like in here?

Reply:
Good idea, I can include a template that when I go through the next release (a few small bug fixes)